README.md

```diff
@@ -1,4 +1,4 @@
-# 🌲 Open Yield Tables
+<img src="https://cdn.tree.ly/assets/v4/yield-tables/logo.svg" alt="Yield Tables logo" width="280"/>
```

Comment:
![Screenshot_20240422-161837.png](https://github.com/treely/openyieldtables/assets/51607747/ecaf5f3c-a2f2-4f5f-8606-54bf9b0c013e)

In dark mode, it's hard to read the logo.

Reply:
Now the logo has a white background.